src/commands/contract/call_function/as_read_only/mod.rs

```diff
@@ -137,29 +138,33 @@ fn call_view_function(
                 function_name, account_id, network_config.network_name
             )
         })?;
-    call_result.print_logs();
-    let mut info_str = String::new();
-    if call_result.result.is_empty() {
-        info_str.push_str("\nEmpty result");
+
+    if let crate::Verbosity::Quiet = verbosity {
```

Comment:
Move that to the 157 line. It can be if else instead of two checks...

Reply:
I disagree with you. Since I'm considering three verbosity options (Quiet, Interactive, TeachMe), it's better not to perform unnecessary operations.

Reply:
To have this printing in one place rather than at the top of functions, and at the end of the functions. It's harder to read the code personally

Reply:
Done